package.json

```diff
@@ -1,46 +1,55 @@
-	"author": {
-		"name": "10up",
-		"email": "opensource@10up.com",
-		"url": "https://10up.com",
-		"role": "developer"
-	},
-	"license": "GPL-2.0-or-later",
-	"main": "build/index.js",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/10up/retro-winamp-block"
-	},
-	"scripts": {
-		"build": "wp-scripts build",
-		"format": "wp-scripts format",
-		"wp-env": "wp-env",
-		"lint:css": "wp-scripts lint-style",
-		"lint:js": "wp-scripts lint-js",
-		"start": "NODE_ENVIRONMENT=development wp-scripts start",
-		"packages-update": "wp-scripts packages-update"
-	},
-	"dependencies": {
-		"@wordpress/blob": "^3.2.2",
-		"@wordpress/block-editor": "^7.0.4",
-		"@wordpress/blocks": "^11.1.2",
-		"@wordpress/components": "^19.0.3",
-		"@wordpress/compose": "^5.0.6",
-		"@wordpress/data": "^6.1.4",
-		"@wordpress/element": "^4.0.4",
-		"@wordpress/i18n": "^4.2.3",
-		"@wordpress/primitives": "^3.0.4",
-		"butterchurn": "^2.6.7",
-		"butterchurn-presets": "^2.4.7",
-		"classnames": "^2.3.1",
-		"lodash": "^4.17.21",
-		"webamp": "^1.5.0"
-	},
-	"devDependencies": {
-		"@wordpress/env": "^5.5.0",
-		"@wordpress/scripts": "^24.3.0"
-	}
+  "name": "winamp-block",
+  "version": "1.2.0",
+  "description": "A Winamp-styled audio block for all your retro music player needs.",
+  "author": {
+    "name": "10up",
+    "email": "opensource@10up.com",
+    "url": "https://10up.com",
+    "role": "developer"
+  },
+  "license": "GPL-2.0-or-later",
+  "main": "build/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/10up/retro-winamp-block"
+  },
+  "scripts": {
+    "build": "wp-scripts build",
+    "format": "wp-scripts format",
+    "wp-env": "wp-env",
+    "env:start": "wp-env start",
+    "env:stop": "wp-env stop",
+    "lint:css": "wp-scripts lint-style",
+    "lint:js": "wp-scripts lint-js",
+    "start": "NODE_ENVIRONMENT=development wp-scripts start",
+    "packages-update": "wp-scripts packages-update",
+    "cypress:open": "cypress open --config-file tests/cypress/config.js --e2e --browser chrome",
+    "cypress:run": "cypress run --config-file tests/cypress/config.js",
+    "env": "wp-env",
+    "postenv:start": "./tests/bin/initialize.sh"
+  },
+  "dependencies": {
+    "@wordpress/blob": "^3.2.2",
+    "@wordpress/block-editor": "^7.0.4",
+    "@wordpress/blocks": "^11.1.2",
+    "@wordpress/components": "^19.0.3",
+    "@wordpress/compose": "^5.0.6",
+    "@wordpress/data": "^6.1.4",
+    "@wordpress/element": "^4.0.4",
+    "@wordpress/i18n": "^4.2.3",
+    "@wordpress/primitives": "^3.0.4",
+    "butterchurn": "^2.6.7",
+    "butterchurn-presets": "^2.4.7",
+    "classnames": "^2.3.1",
+    "lodash": "^4.17.21",
+    "webamp": "^1.5.0"
+  },
+  "devDependencies": {
+    "@10up/cypress-wp-utils": "github:10up/cypress-wp-utils#build",
+    "@wordpress/env": "^5.12.0",
+    "@wordpress/scripts": "^24.3.0",
+    "cypress": "^10.11.0",
+    "cypress-file-upload": "^5.0.8"
```

Comment:
It seems from Cypress [9.3.0](https://docs.cypress.io/guides/references/changelog?ref=cypress-blog#9-3-0), it already have [`.selectFile()`](https://docs.cypress.io/api/commands/selectfile) command which has similar API to  cypress-file-upload. Can we remove this package if it is no longer needed (I didn't find usage in this PR)?